tfprotov5/resource.go

```diff
@@ -145,7 +145,7 @@ type UpgradeResourceIdentityRequest struct {
 	// RawIdentity is the identity state as Terraform sees it right now. See the
 	// documentation for `RawIdentity` for information on how to work with the
 	// data it contains.
-	RawIdentity *RawIdentity
+	RawState *RawState
```

Comment:
We should keep this matching the protocol, which is still named `raw_identity`

```suggestion
	RawIdentity *RawState
```